### ISSUE TITLE
Could you add support for swift?

### ISSUE THREAD

**Issue by MellongLau** [lucholaf/Auto-Importer-for-Xcode#18]
You know, more and more developer move to swift, could you add support for swift?

**Comment by lucholaf** [comment]
@MellongLau what you mean? There is no need to import specific class/headers files on swift.

**Comment by psophis** [comment]
What about for Frameworks?

**Comment by mathmatrix828** [comment]
+1 for importing libs, like those from Cocoapods

**Comment by yaroslavyaroslav** [comment]
Yep, it would be greate if it will be some swift mode (at least with manual swith). Which cocoapods will import not like now "#import "MBProgressHUD.h", but like that: "import "MBProgressHUD".

Everything else works well (plugin finds cocapods for swift as well - the only problem is syntax.
